### PR TITLE
Change opentracing_fat from jsonj to jsonp11

### PR DESCRIPTION
**Pull request by tbitonti**
Should correct the dependency.

### CODE REVIEW

**Comment by tbitonti** [comment]
For issue 386: Re-enable open tracing FAT
https://github.com/OpenLiberty/open-liberty/issues/386

**Comment by tbitonti** [comment]
#build

File: dev/com.ibm.ws.opentracing_fat/.classpath [hunk 1]
```diff
@@ -4,5 +4,6 @@
 	<classpathentry kind="src" path="test-applications/serviceApp/src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.jsonp.1.1"/>
```
Comment (aguibert) [inline dev/com.ibm.ws.opentracing_fat/.classpath]: This should not be here -- all of this should get delegated to the bnd.bnd file

**Comment by tbitonti** [comment]
#build

**Comment by tbitonti** [comment]
#build

**Pull request merged**